pkg/porter/build_test.go

```diff
@@ -121,3 +121,34 @@ func TestPorter_paramRequired(t *testing.T) {
 	require.False(t, bundle.Parameters["command"].Required, "expected command param to not be required")
 	require.True(t, bundle.Parameters["command2"].Required, "expected command2 param to be required")
 }
+
+func TestValidateBuildOpts(t *testing.T) {
+	testcases := []struct {
+		name      string
+		opts      BuildOptions
+		wantError string
+	}{{
+		name:      "no opts",
+		opts:      BuildOptions{},
+		wantError: "",
+	}, {
+		name:      "invalid version set",
+		opts:      BuildOptions{metadataOpts: metadataOpts{Version: "latest"}},
+		wantError: `invalid bundle version "latest".  Cannot be parsed as semver: Invalid Semantic Version`,
+	}, {
+		name:      "valid name and value set",
```

Comment:
Let's add one more test case for "v1.0.0", which should fail.

Reply:
Can do, pending resolution on https://github.com/getporter/porter/pull/1368#discussion_r528882681

Reply:
Test added!